src/table/context/createSelectorProvider.tsx

```diff
@@ -43,7 +43,7 @@ export function createSelectorProvider<T>(OriginalContext: Context<T>): Function
       return contextValueRef.current;
     }, [contextValueRef]);
 
-    const contextValue: SelectorContextType<T> = useMemo(() => [getContextValue, listeners.current], [contextValueRef]);
+    const contextValue: SelectorContextType<T> = useMemo(() => [getContextValue, listeners.current], [getContextValue]);
```

Comment:
Hmm I would think that if not, `contextValue` would always be a new array. Possibly triggering a new "render" because the provider value would change every time `createSelectorProvider` is called.